AllReadyApp/Web-App/AllReady/Areas/Admin/ViewModels/Validators/Task/TaskEditViewModelValidator.cs

```diff
@@ -6,24 +6,29 @@
 using AllReady.Models;
 using MediatR;
 using System.IO;
+using AllReady.Services;
 
 namespace AllReady.Areas.Admin.ViewModels.Validators.Task
 {
     public class TaskEditViewModelValidator : ITaskEditViewModelValidator
     {
-        private static readonly IList<string> AllowedFileExtensions = new List<string> { ".png", ".jpg", ".doc", ".docx", ".xls", ".xlsx", ".pdf" };
-        private const int MaxAttachmentBytes = 2 * 1024 * 1024; // 2MB
-
         private readonly IMediator _mediator;
+        private readonly IAttachmentService _attachmentService;
 
-        public TaskEditViewModelValidator(IMediator mediator)
+        public TaskEditViewModelValidator(IMediator mediator, IAttachmentService attachmentService)
         {
             if (mediator == null)
```

Comment:
please delete these constructor guard clauses. We don't really use constructor guard clauses because 
1. at runtime if an instance of a dependency is missing, an argument will be thrown anyhow, and that's the sign of mis-configuration somewhere
2. it requires unit tests to provide mocks of dependencies whose api might not be executed in the given unit test.

thanks.